io.github.leolost2605.emergency-alerts.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.github.leolost2605.emergency-alerts
+
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+
+command: io.github.leolost2605.emergency-alerts
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.gtk.vfs.*'
+  - '--filesystem=xdg-run/gvfsd'
+
+modules:
+  - name: sassc
+    cleanup:
+      - '*'
+    sources:
+      - type: git
+        url: https://github.com/sass/sassc.git
+        tag: '3.6.2'
+
+      - type: script
+        dest-filename: "autogen.sh"
+        commands:
+          - autoreconf -si
+    modules:
+      - name: libsass
+        cleanup:
+          - '*'
+        sources:
+          - type: git
+            url: https://github.com/sass/libsass.git
+            tag: '3.6.5'
+          - type: script
+            dest-filename: "autogen.sh"
+            commands:
+              - autoreconf -si
+
+  - name: granite-7
+    buildsystem: meson
+    config-opts:
+      - -Ddemo=false
+    sources:
+      - type: git
+        url: https://github.com/elementary/granite.git
+        tag: '7.6.0'
+        commit: 065ba1f245b60ef867a6ca145e0891cf84eddcc7
```

Comment:
maybe the elementary baseapp should be updated/ new branch should be created with this, not a blocker though as that's low in maintenance.

Reply:
Tbh I didn't even know there was a baseapp for elementary
Seems like there's already a new branch in place and a PR for adding granite 7: flathub/io.elementary.BaseApp#30

Reply:
@bbhtt I think I'd prefer to go with it as is for now since the BaseApp seems to have somewhat stalled with the maintainer not being very active (> 7 months) and I'm not sure I really want to rely on it :grimacing: 
I'll see about contacting the maintainer regarding transferring/taking over maintainership and would just update it then if that's ok :)

Reply:
> maintainer not being very active (> 7 months), 

You meant "3 weeks" ?

https://github.com/flathub/io.elementary.BaseApp/commits/branch/circe-24.08

Reply:
you can also contribute. that's how things work.

Reply:
@hfiguiere I'm so sorry I fear I sounded much more harsh than intended (not sounding harsh at all) :/
By no means I wanted to criticize anyone, of course I understand people have limited time they want rather spend in other areas and I'm thankful to everyone here. I was just trying to lay out reasons why I'd rather bundle the dependencies myself for now and thereby use one of the great features of flatpak - having full control over your dependencies.

> You meant "3 weeks" ?

I saw that but I thought you were a flathub admin who was friendly enough to take a look at some branches that didn't receive much attention lately, I'm sorry if I misunderstood something there :/

> you can also contribute. that's how things work.

The things required had already a PR (https://github.com/flathub/io.elementary.BaseApp/pull/30) open for them which seemed to be ready so I didn't think I could do much :/